frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx

```diff
@@ -38,22 +36,12 @@ const CommunityAccordion: React.FunctionComponent<CommunityExperience> = ({
     <Accordion
       title={intl.formatMessage(
         {
-          defaultMessage: "{title} at {organization}",
+          defaultMessage: "{title} of {organization}",
```

Comment:
Was this change intentional?  I'm having trouble finding Jerbo's post that had all the individual accordions.

Reply:
Found it!  Community should be _at_ not _of_.  https://github.com/GCTC-NTGC/gc-digital-talent/issues/1925#issuecomment-1054640559

Reply:
I don't think I change it.

Reply:
You changed it in commit https://github.com/GCTC-NTGC/gc-digital-talent/pull/2454/commits/abb62a0633834bc3788d761179cb52d2adb549cf .  Hehe, git knows everything!

Reply:
really? why I did that?

Reply:
I changed it back.